lib/dev.cfg

```diff
@@ -20,15 +20,19 @@ model = {
     # name = "smc_prime",
 
     # The 'Dirac' multiple mergers coalescent
-    population_size = 1000.0;
-    name = "dirac",
-    psi = 0.99;
-    c = 0.1;
+    # population_size = 1000.0;
+    # name = "dirac",
+    # psi = 0.99;
+    # c = 0.1;
 
     # The 'Beta' multiple mergers coalescent
     #name = "beta";
     #alpha = 2.2;
     #truncation_point = 10.0;
+
+    # The discrete time Wright Fisher
+    name = "wright_fisher";
```

Comment:
Let's make this 'dtwf' for consistency. (We might want to put in a forwards time wright fisher in the future, for example)